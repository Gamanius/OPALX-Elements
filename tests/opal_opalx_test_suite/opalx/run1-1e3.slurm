#!/bin/bash
#SBATCH --partition=hourly      # Using 'hourly' will grant higher priority
#SBATCH --nodes=1               # No. of nodes
#SBATCH --ntasks-per-node=1     # No. of MPI ranks per node. Merlin CPU nodes have 44 cores
#SBATCH --cpus-per-task=1      # No. of OMP threads
#SBATCH --time=00:30:00         # Define max time job will run (e.g. here 5 mins)
#SBATCH --hint=nomultithread    # Without hyperthreading
##SBATCH --exclusive            # The allocations will be exclusive if turned on (remove extra hashtag to turn on)

#SBATCH --output=run1-1e3.in.out  # Name of output file
#SBATCH --error=run1-1e3.in.err    # Name of error file

export OMP\_NUM\_THREADS=1
export OMP\_PROC\_BIND=spread
export OMP\_PLACES=threads


srun --cpus-per-task=1 /data/user/binder_j/opalx-test/build/src/opalx run1-1e3.in --info 10
