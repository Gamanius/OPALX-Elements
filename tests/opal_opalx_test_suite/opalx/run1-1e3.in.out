Warning: Option 'run1-1e3.in' is not parsed by Ippl.
OPAL-X> configure done argc= 4
OPAL-X>               ____  _____        _       __   __
OPAL-X> 
OPAL-X> This is OPAL-X (Object Oriented Parallel Accelerator Library) Version 2024.1.00
OPAL-X>             git rev. ac1b7f5951630e232502575123a79f799192cf6e
OPAL-X> 
OPAL-X> 
OPAL-X>                      (c) PSI, http://amas.web.psi.ch
OPAL-X> 
OPAL-X> 
OPAL-X> This is the performance portable version of OPAL
OPAL-X> 
OPAL-X> Please send cookies, goodies or other motivations (wine and beer ... ) 
OPAL-X> to the OPAL developers opal@lists.psi.ch
OPAL-X> 
OPAL-X> Time: 14:04:59 date: 20/05/2025
OPAL-X> 
OPAL-X> * Host:   OpenMP
OPAL-X> * Device: OpenMP
OPAL-X> 
OPAL-X> * Reading input stream 'run1-1e3.in'
OPAL-X> 
OPAL-X> value: {EDES,P0,OPALVERSION} = {0.001, 0.00142197, 2.02401e+07}
OPAL-X> 
OPAL-X> * Number of distributions  1
OPAL-X> 
OPAL-X> * ************* D I S T R I B U T I O N ********************************************
OPAL-X> * 
OPAL-X> * Distribution type: GAUSS
OPAL-X> * 
OPAL-X> * SIGMAX     = 0.001 [m]
OPAL-X> * SIGMAY     = 0.001 [m]
OPAL-X> * SIGMAZ     = 0.001 [m]
OPAL-X> * SIGMAPX    = 0 [Beta Gamma]
OPAL-X> * SIGMAPY    = 0 [Beta Gamma]
OPAL-X> * SIGMAPZ    = 0 [Beta Gamma]
OPAL-X> * 
OPAL-X> * Distribution is injected.
OPAL-X> * 
OPAL-X> * **********************************************************************************
OPAL-X> 
OPAL-X> * ************* F I E L D S O L V E R ********************************************** 
OPAL-X> * FIELDSOLVER  FS1
OPAL-X> * TYPE         NONE
OPAL-X> * RANKS       1
OPAL-X> * NX           16
OPAL-X> * NY           16
OPAL-X> * NZ           16
OPAL-X> * BBOXINCR     1
OPAL-X> * GREENSF      INTEGRATED
OPAL-X> * XDIM         serial  
OPAL-X> * YDIM         serial  
OPAL-X> * ZDIM      parallel  
OPAL-X> * ********************************************************************************** 
OPAL-X> 
OPAL-X> * ************* B E A M ************************************************************ 
OPAL-X> * BEAM        BEAM1
OPAL-X> * PARTICLE    ELECTRON
OPAL-X> * REST MASS   0.000510999 [GeV]
OPAL-X> * CHARGE      -e * 1 
OPAL-X> * MOMENTUM    1.42197e+06 [eV/c]
OPAL-X> * MOMENTUM    0.00142197 [GeV/c]
OPAL-X> * CURRENT     1.3 [A]
OPAL-X> * FREQUENCY   1300 [MHz]
OPAL-X> * NPART       1000
OPAL-X> * ********************************************************************************** 
OPAL-X> 
OPAL-X> PartBunch Constructor
OPAL-X> * Solver and Load Balancer set
OPAL-X> number_of_processors 88
OPAL-X> MPI_Comm_size 1
OPAL-X> * About to create particles ...
OPAL-X> * Particle creation done
OPAL-X> 
OPAL-X> * ************** B U N C H ********************************************************* 
OPAL-X> * PARTICLES       = 1000
OPAL-X> * CHARGE          = -1.000000e-09 (Cb) 
OPAL-X> * INTEGRATOR      = LF2
OPAL-X> * MIN R (origin)  = (     -2.93326 ,     -2.78186 ,     -2.97862 ) [mm]
OPAL-X> * MAX R (max ext) = (      2.92717 ,      2.85236 ,      2.96385 ) [mm]
OPAL-X> * RMS R           = (      0.98388 ,      0.96069 ,      1.00555 ) [mm]
OPAL-X> * RMS P           = ( 0.000000e+00 , 0.000000e+00 , 2.930989e-14 ) [beta gamma]
OPAL-X> * Mean R: ( 7.806256e-21 , 3.838076e-20 , -8.131516e-21 ) [m]
OPAL-X> * Mean P: ( 0.000000e+00 , 0.000000e+00 , 2.782725e+00 ) [beta gamma]
OPAL-X> * MESH SPACING    = (    369.93965 ,    355.66001 ,    375.11842 ) [um]
OPAL-X> * COMPDOM INCR    = 1.000000e+00 (%) 
OPAL-X> * FIELD LAYOUT    = Domain = {[0:15:1],[0:15:1],[0:15:1]}
OPAL-X> Total number of boxes = 1
OPAL-X>     Box 0 {[0:15:1],[0:15:1],[0:15:1]}
OPAL-X> 
OPAL-X> * Centroid : 
OPAL-X> * 7.806256e-18 0.000000e+00 3.838076e-17 0.000000e+00 -8.131516e-18 2.782725e+03 
OPAL-X> * Cov Matrix : 
OPAL-X> * 9.680198e-07 0.000000e+00 2.104185e-08 0.000000e+00 6.510467e-09 1.479114e-34 
OPAL-X> * 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 
OPAL-X> * 2.104185e-08 0.000000e+00 9.229233e-07 0.000000e+00 -1.804035e-08 1.254166e-33 
OPAL-X> * 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 
OPAL-X> * 6.510467e-09 0.000000e+00 -1.804035e-08 0.000000e+00 1.011136e-06 -6.070531e-34 
OPAL-X> * 1.479114e-34 0.000000e+00 1.254166e-33 0.000000e+00 -6.070531e-34 8.590695e-28 
OPAL-X> * * ********************************************************************************** 
OPAL-X> * Creating new file for data: 'run1-1e3.stat'
OPAL-X> * Creating new file for data: 'run1-1e3.lbal'
OPAL-X> * BF gone live
OPAL-X> ParallelTrack: momentum=  2.78273
OPAL-X> itsBunch_m->RefPartR_m= ( 7.80626e-21 , 3.83808e-20 , -8.13152e-21 )
OPAL-X> itsBunch_m->RefPartP_m= ( 0 , 0 , 2.78273 )
OPAL-X> OrbitThreader dt_m= 1e-10
OPAL-X> OrbitThreader maxDistance= 1.79769e+308
OPAL-X> OrbitThreader #elements  = 1
OPAL-X> OrbitThreader maxDistance= 1.28996
OPAL-X> OrbitThreader #elements  = 0
OPAL-X> 
OPAL-X> * Size of map 2 sections 
OPAL-X> * Key: (      0.000000 -       0.507831) number of overlapping elements 1
OPAL-X> *                                        BF
OPAL-X> * Key: (      0.507831 - 1000006.000000) number of overlapping elements 0
OPAL-X> 
OPAL-X> * Creating new file for data: 'data/run1-1e3_ElementPositions.sdds'
OPAL-X> * Track start at: 14:04:59, t= 0.000 [fs]; zstart at: 0.000 [um]
OPAL-X> * Executing ParallelTracker
OPAL-X> * Initial dt = 100.000 [ps]
OPAL-X> * Max integration steps = 33, next step = 0
OPAL-X> 
OPAL-X>                dt [ns]           zStop [m]        num Steps [1]
OPAL-X> ->         1.000000e-10        1.000000e+06                  33
OPAL-X> no solver avaidable 
OPAL-X> * 14:04:59 Step      0 at 28.213 [mm], t= 100.000 [ps], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:04:59 Step      1 at 56.426 [mm], t= 200.000 [ps], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:04:59 Step      2 at 84.639 [mm], t= 300.000 [ps], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:04:59 Step      3 at 112.851 [mm], t= 400.000 [ps], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:04:59 Step      4 at 141.064 [mm], t= 500.000 [ps], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:04:59 Step      5 at 169.277 [mm], t= 600.000 [ps], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:04:59 Step      6 at 197.490 [mm], t= 700.000 [ps], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:04:59 Step      7 at 225.703 [mm], t= 800.000 [ps], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step      8 at 253.916 [mm], t= 900.000 [ps], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step      9 at 282.128 [mm], t= 1.000 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     10 at 310.341 [mm], t= 1.100 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     11 at 338.554 [mm], t= 1.200 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     12 at 366.767 [mm], t= 1.300 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     13 at 394.980 [mm], t= 1.400 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     14 at 423.193 [mm], t= 1.500 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     15 at 451.406 [mm], t= 1.600 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     16 at 479.618 [mm], t= 1.700 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     17 at 507.831 [mm], t= 1.800 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     18 at 536.044 [mm], t= 1.900 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     19 at 564.257 [mm], t= 2.000 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     20 at 592.470 [mm], t= 2.100 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     21 at 620.683 [mm], t= 2.200 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     22 at 648.895 [mm], t= 2.300 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     23 at 677.108 [mm], t= 2.400 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     24 at 705.321 [mm], t= 2.500 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     25 at 733.534 [mm], t= 2.600 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     26 at 761.747 [mm], t= 2.700 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     27 at 789.960 [mm], t= 2.800 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     28 at 818.173 [mm], t= 2.900 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     29 at 846.385 [mm], t= 3.000 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     30 at 874.598 [mm], t= 3.100 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     31 at 902.811 [mm], t= 3.200 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> no solver avaidable 
OPAL-X> * 14:05:00 Step     32 at 931.024 [mm], t= 3.300 [ns], E=1.000 [MeV]
OPAL-X> * Wrote beam statistics.
OPAL-X> * Dump phase space of last step
OPAL-X> * BF gone off
OPAL-X> 
OPAL-X> * Done executing ParallelTracker at 14:05:00
OPAL-X> 
OPAL-X> * Finished time step: 0 time: 0.000000e+00
OPAL-X> * End of input stream 'run1-1e3.in'
Timings> ---------------------------------------------
Timings>      Timing results for 1 nodes:
Timings> ---------------------------------------------
Timings> mainTimer........... Wall tot =    1.09359
Timings> 
Timings> gatherInfoPartBunch. Wall max = 0.000944599
Timings>                      Wall avg = 0.000944599
Timings>                      Wall min = 0.000944599
Timings> 
Timings> setSolver........... Wall max = 0.000195679
Timings>                      Wall avg = 0.000195679
Timings>                      Wall min = 0.000195679
Timings> 
Timings> prerun.............. Wall max =  4.739e-05
Timings>                      Wall avg =  4.739e-05
Timings>                      Wall min =  4.739e-05
Timings> 
Timings> samplingTime........ Wall max = 0.00125871
Timings>                      Wall avg = 0.00125871
Timings>                      Wall min = 0.00125871
Timings> 
Timings> SamplingTimer....... Wall max = 0.00117312
Timings>                      Wall avg = 0.00117312
Timings>                      Wall min = 0.00117312
Timings> 
Timings> GenParticles........ Wall max = 0.00117589
Timings>                      Wall avg = 0.00117589
Timings>                      Wall min = 0.00117589
Timings> 
Timings> particleBC.......... Wall max = 3.0072e-05
Timings>                      Wall avg = 3.0072e-05
Timings>                      Wall min = 3.0072e-05
Timings> 
Timings> updateParticle...... Wall max =          0
Timings>                      Wall avg =          0
Timings>                      Wall min =          0
Timings> 
Timings> Write Stat.......... Wall max = 0.00695589
Timings>                      Wall avg = 0.00695589
Timings>                      Wall min = 0.00695589
Timings> 
Timings> Write H5-File....... Wall max =   0.869592
Timings>                      Wall avg =   0.869592
Timings>                      Wall min =   0.869592
Timings> 
Timings> TIntegration1....... Wall max = 0.000374039
Timings>                      Wall avg = 0.000374039
Timings>                      Wall min = 0.000374039
Timings> 
Timings> TIntegration2....... Wall max = 0.00164836
Timings>                      Wall avg = 0.00164836
Timings>                      Wall min = 0.00164836
Timings> 
Timings> External field eval. Wall max =  0.0263958
Timings>                      Wall avg =  0.0263958
Timings>                      Wall min =  0.0263958
Timings> 
Timings> Binaryrepart........ Wall max =          0
Timings>                      Wall avg =          0
Timings>                      Wall min =          0
Timings> 
Timings> OrbThreader......... Wall max =  0.0348992
Timings>                      Wall avg =  0.0348992
Timings>                      Wall min =  0.0348992
Timings> 
Timings> ---------------------------------------------
